### DISCUSSION TITLE
backlink shortcut on status bar

### DISCUSSION THREAD

**Discussion by timelyfiasco** [zim-desktop-wiki/zim-desktop-wiki#1674] [Development]
Hi, version 0.74 doesnt have the status bar anymore. I find the alt+b shortcut for backlinks useful. Is it permanently gone?

**Comment by jaap-karssenberg** [comment]
It is gone as a core feature. Idea is that the backlinks side pane plugin
is a good replacement. Could make sense to have a shortcut for bringing up
the side pane.

Op zo 10 okt. 2021 05:11 schreef timelyfiasco ***@***.***>:

**Comment by timelyfiasco** [comment]
Thanks jaap, I'll wait for that.

**Comment by jaap-karssenberg** [comment]
Added mnemonic in e0961b1d2da9861bbff9711744dfce2427b3fe08 so Alt-B no switches to the backlink pane
